### PR TITLE
NullLogger has relocated

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerActionDiscoveryTest.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 using Microsoft.AspNet.Mvc.Core;
 using Microsoft.AspNet.Mvc.Filters;
 using Microsoft.AspNet.Mvc.WebApiCompatShim;
-using Microsoft.Framework.Logging;
+using Microsoft.AspNet.Testing.Logging;
```
Comment (dougbu) [inline test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerActionDiscoveryTest.cs]: nit: one of the `NullLogger` mentions in this file uses `new NullLogger()` while the reset use `NullLogger.Instance`.  let's be consistent and use `Instance` everywhere (just one case to change).

**Comment by dougbu** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:shipit:

**Comment by BrennanConroy** [comment]
Should we still do this since Pranav changed the tests to use new LoggerFactory()

**Comment by NTaylorMullen** [comment]
Hmm well we have the test class already, lets keep it as is, makes it blatantly obvious what it's being used for.

**Comment by dougbu** [comment]
@BrennanConroy suggest closing this PR unless you're planning to do something with it

**Pull request closed without merging**